dhis-2/dhis-support/dhis-support-db-migration/src/main/java/org/hisp/dhis/db/migration/v34/V2_34_22__Add_teav_btree_index.java

```diff
@@ -0,0 +1,44 @@
+package org.hisp.dhis.db.migration.v34;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class V2_34_22__Add_teav_btree_index
```

Comment:
Add class javadoc about what this migration is doing and the "special case" of catching failing index creation.